### PR TITLE
Adds abbr, address, code, hr, keyboard and variable styled elements t…

### LINKED ISSUE #346: Abbreviation
>The HTML Abbreviation element (`<abbr>`) represents an abbreviation or acronym; the optional title attribute can provide an expansion or description for the abbreviation.

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
Fixes #346  #347 #353 #371 #367 #409 and #447

Adds `abbr`, `address`, `code`, `hr`, `kbd` and `var` to glossary page. Also alphabetizes all glossary items.

**Note:** `address` tag is reflected in the glossary but has no style changes/overrides. It will be displayed as any other default type would.

### CODE REVIEW

File: packages/docs/source/changelog.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!-- # Changelog
```
Comment (edburyenegren-okta) [inline packages/docs/source/changelog.md:1]: Looks like this slipped in!

File: packages/odyssey/src/scss/base/_typography-text.scss [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+  > em {
+    text-decoration: underline;
+  }
 }
 
 strong {
   font-weight: 600;
 
-  > strong { text-decoration: underline; }
+  > strong {
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/base/_typography-text.scss:17]: Are nested `strong`s allowed/intended in the same way as `em`?

File: packages/odyssey/src/scss/base/_typography-text.scss [hunk 3]
```diff
@@ -80,8 +86,39 @@ ins::after {
...
+
+s {
+  text-decoration: line-through;
+}
+
+small {
+  font-size: ms(-1);
+}
+
+kbd {
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/base/_typography-text.scss:101]: do we wanna use `$mono-font-family` here?

File: packages/odyssey/src/scss/base/_typography-text.scss [hunk 4]
```diff
@@ -80,8 +86,39 @@ ins::after {
...
+  padding: 0 $tiny-spacing;
+  border: 1px solid cv('gray', '200');
+  border-radius: $base-border-radius;
+  background: cv('gray', '000');
+  box-shadow: 0 2px 0 cv('gray', '000'), 0 3px 0 cv('gray', '200');
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/base/_typography-text.scss:106]: love this.

File: packages/docs/source/building-blocks/elements.md [hunk 5]
```diff
@@ -2,6 +2,43 @@
...
+</figure>
+
+## address <a name="address"></a> 
+> The HTML `<address>` element indicates that the enclosed HTML provides contact information for a person or people, or for an organization. - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <p>Contact the author of this page:</p>
+
+    <address>
```
Comment (edburyenegren-okta) [inline packages/docs/source/building-blocks/elements.md:26]: This is rendering weird for me:

Screenshot 2020-04-06 10 11 42

File: packages/odyssey/src/scss/base/_typography-text.scss (hunk 2 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/base/_typography-text.scss:17]: They sure can!

See: [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong](MDN <strong>: The Strong Importance element) > Em vs Strong

File: packages/odyssey/src/scss/base/_typography-text.scss (hunk 3 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey/src/scss/base/_typography-text.scss:101]: image

Was going to use it here, but decided against. Our default text flows much better inline with other copy.

File: packages/docs/source/building-blocks/elements.md (hunk 5 above)
Comment (arnoldsandoval-okta) [reply packages/docs/source/building-blocks/elements.md:26]: Fixed this, had to do with spacing in markdown which affected the output

**Pull request merged**